src/help-generator_.toit

```diff
@@ -531,8 +532,21 @@ class HelpGenerator:
     // Reconstruct the full command line, but now with the options next to the
     // commands that defined them.
     full-command := []
+    is-root := true
+    // For examples we don't want the full path that was used to invoke the
+    // command (like `build/bin/artemis`), but only the basename.
+    app-name := invoked-command_
```

Comment:
Should we have a basename of sorts somewhere?

Reply:
added a local `basename_` function.
done.